aiida/storage/sqlite_temp/backend.py

```diff
@@ -51,7 +51,8 @@ class Configuration(BaseModel):
             default_factory=mkdtemp
         )
 
-    _read_only = False
+    read_only = False
+    """This plugin is read only and data cannot be created or mutated."""
```

Comment:
Did you want to add this comment to the `SqliteZipBackend` instead?

As a side-note, I'm curious about this use of triple double quotes for adding a comment below a class attribute. Does it have any special effect compared to using `#`?

Reply:
It's just the recommended style for attribute docstrings from [PEP 258](https://peps.python.org/pep-0258/#attribute-docstrings).

Reply:
And yes, this was intended for the `core.sqlite_zip` one, good catch